examples/protocol_buffers/BUILD.bazel

```diff
@@ -55,10 +55,48 @@ ts_devserver(
 )
 
 # Test for production mode
-load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle")
+load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle", "nodejs_binary")
 
 rollup_bundle(
     name = "bundle",
     deps = [":app"],
     entry_point = "examples/protocol_buffers/app",
+    globals = {
```

Comment:
this map is already defined in the `js_library` - any idea how we could re-use it?

Reply:
okay let's not wait too long :)